__tests__/search.ts

```diff
@@ -528,3 +529,121 @@ describe('query searchSourceSuggestions', () => {
     ]);
   });
 });
+
+describe('query searchUserSuggestions', () => {
+  const QUERY = (query: string): string => `{
+    searchUserSuggestions(query: "${query}") {
+      query
+      hits {
+        id
+        title
+        subtitle
+        image
+      }
+    }
+  }
+`;
+
+  beforeEach(async () => {
+    await saveFixtures(con, User, usersFixture);
+  });
+
+  it('should return search suggestions', async () => {
+    const res = await client.query(QUERY('i'));
+    expect(res.errors).toBeFalsy();
+    expect(res.data.searchUserSuggestions).toBeTruthy();
+
+    const result = res.data.searchUserSuggestions;
+
+    expect(result.query).toBe('i');
+    expect(result.hits).toHaveLength(3);
+    expect(result.hits).toMatchObject([
+      {
+        id: '1',
+        image: 'https://daily.dev/ido.jpg',
+        subtitle: 'idoshamun',
+        title: 'Ido',
+      },
+      {
+        id: '2',
+        image: 'https://daily.dev/tsahi.jpg',
+        subtitle: 'tsahidaily',
+        title: 'Tsahi',
+      },
+      {
+        id: '3',
+        image: 'https://daily.dev/nimrod.jpg',
+        subtitle: 'nimroddaily',
+        title: 'Nimrod',
+      },
+    ]);
+  });
+
+  it('should only return infoConfirmed users', async () => {
+    await con.getRepository(User).update({ id: '3' }, { infoConfirmed: false });
+    const res = await client.query(QUERY('i'));
+    expect(res.data.searchUserSuggestions).toBeTruthy();
+
+    const result = res.data.searchUserSuggestions;
+
+    expect(result.query).toBe('i');
+    expect(result.hits).toHaveLength(2);
+    expect(result.hits).toMatchObject([
+      {
+        id: '1',
+        image: 'https://daily.dev/ido.jpg',
+        subtitle: 'idoshamun',
+        title: 'Ido',
+      },
+      {
+        id: '2',
+        image: 'https://daily.dev/tsahi.jpg',
+        subtitle: 'tsahidaily',
+        title: 'Tsahi',
+      },
+    ]);
+  });
+
+  it('should only return vodr false users', async () => {
+    await con.getRepository(User).update(
+      { id: '3' },
+      {
+        flags: updateFlagsStatement<User>({
+          vordr: true,
+        }),
+      },
+    );
```

Comment:
Could use the user `vordr` from `badUsersFixture`?

Reply:
Ah think this is easier as we can then test again the succesfull test above if that's ok?